mismi-s3/test/Test/Reliability/Mismi/S3/Commands.hs

```
@@ -28,9 +28,11 @@ import           Test.Reliability.Reliability
 import           Test.QuickCheck
 import           Test.QuickCheck.Instances ()
 
+import           X.Control.Monad.Trans.Either (runEitherT)
+
 prop_sync = forAll (elements muppets) $ \m -> testAWS' $ \a b i -> do
   createFiles a m i
-  syncWithMode OverwriteSync a b 10
+  _ <- runEitherT $ syncWithMode OverwriteSync a b 10
```

Ignoring errors? Should we at least fail on left?